com.unity.multiplayer.mlapi/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -162,6 +162,20 @@ internal static void OnSceneSwitch(uint sceneIndex, Guid switchSceneGuid, Stream
             }
         }
 
+        public static uint GetCurrentSceneIndex()
+        {
+            Scene activeScene = SceneManager.GetActiveScene();
+            if(sceneNameToIndex.Count == 0)
+            {
+                return (uint)activeScene.buildIndex;
+            }
+            else
+            {
+                string sceneName = SceneManager.GetActiveScene().name;
+                return sceneNameToIndex[sceneName];
+            }
+        }
+
```

Comment:
why we need this?

Reply:
I don't think we do need it.  Certainly not for RPC Queue stuff.

Reply:
Removed this code.